how-to-guides/cloud-services/gcp/connecting-to-secret-manager.md

```diff
@@ -0,0 +1,177 @@
+# GCP Secret Manager
+
+You are connecting to a GCP Secret Manager from your DevBox.
+
+## Architecture Diagram
+
+![GCP Key Vault Architecture](../../../.gitbook/assets/gcp-secret-manager-architecture.png)
+
+Here, you will connect to a GCP Secret Manager from your DevBox. This can be done using any programming language API, but for this demo, we will use Python.
```

Comment:
```suggestion
Here, you will connect to a GCP Secret Manager from your DevBox. This can be done using any programming language API, but for the scope of this document, we will use Python.
```